handlers/graph.go

```diff
@@ -63,7 +63,10 @@ func GraphNamespaces(
 	return func(w http.ResponseWriter, r *http.Request) {
 		defer handlePanic(r.Context(), w)
 
-		// TODO: getLayer needs to extract the logger from r - and probably put it into a Context ??
+		// prepare the logger in a context, and replace the request context with ours that has our logger in it
+		r = log.AddGroupToLoggerInRequestContext(r, log.GraphNamespaceLogName)
```

Comment:
This duplicates some info: `RequestID=d0f27168bj6c73caki4g group=graphNamespace route=GraphNamespaces`

Namely: `group=graphNamespace` and `route=GraphNamespaces` convey the same thing.